config/env_all.yaml

```diff
@@ -0,0 +1,282 @@
+database_url:
+  doc: database_url
+  default: postgres://postgres:oiPorg3Nrz0yqDLE@postgres:5432/polis-dev
+  env: DATABASE_URL
+  
+webserver_username:
+  doc: webserver_username
+  default: ws-user
+  env: WEBSERVER_USERNAME
+  
+webserver_pass:
+  doc: webserver_pass
+  default: ws-passPOSTGRES_DB=polis-dev
+  env: WEBSERVER_PASS
+  
+postgres_password:
+  doc: postgres_password
+  default: oiPorg3Nrz0yqDLE
+  env: POSTGRES_PASSWORD
```

Comment:
Generally, can this format be processed to be less redundant? Like why not:

```yaml
POSTGRES_PASSWORD:
  default: oiPorg3Nrz0yqDLE

# Generates JS config object key like:
# POSTGRES_PASSWORD: {
#   default: "oiPorg3Nrz0yqDLE",
#   env: "POSTGRES_PASSWORD",
#   doc: "POSTGRES_PASSWORD",
# }
```

And use in code like:

```js
config.get('POSTGRES_PASSWORD')
```

Would be great if it auto-generated keys (like "doc") if unspecified, but specifying would override:

```yml
POSTGRES_PASSWORD:
  default: oiPorg3Nrz0yqDLE
  doc: Some other doc string.

# Generates JS config object key like:
# POSTGRES_PASSWORD: {
#   default: "oiPorg3Nrz0yqDLE",
#   env: "POSTGRES_PASSWORD",
#   doc: "some other doc string.",
# }
```